digits/utils/image.py

```diff
@@ -378,3 +504,75 @@ def get_color_map(name):
         greenmap    = [0,0,0.5,1,1,1,0.5,0,0]
         bluemap     = [0.5,1,1,1,0.5,0,0,0,0]
     return 255.0 * np.array(redmap), 255.0 * np.array(greenmap), 255.0 * np.array(bluemap)
+
+def rgb_to_hsv(rgb):
+    # Translated from source of colorsys.rgb_to_hsv
+    # r,g,b should be a numpy arrays with values between 0 and 255
+    # rgb_to_hsv returns an array of floats between 0.0 and 1.0.
```

Comment:
It looks like you're getting this code from [this SO post](http://stackoverflow.com/a/7274986/2404152). SO licenses user contributions with a viral license.

> user contributions licensed under cc by-sa 3.0 with attribution required

http://creativecommons.org/licenses/by-sa/3.0/

Unfortunately, that means you're going to need to re-implement this yourself or get permission from the original author to use this code :/